lib/ansible/modules/cloud/ovirt/ovirt_vnic_profile.py

```diff
@@ -67,6 +67,7 @@
         description:
             - "Enables passthrough to an SR-IOV-enabled host NIC."
             - "When enabled C(qos) and  C(network_filter) are automatically set to None and C(port_mirroring) to False."
+            - "When enabled and C(qos) or C(network_filter) or C(port_mirroring) specified module will raise exeption."
```

Comment:
Please remove and write: `Port mirroring, QoS and network filters are not supported on passthrough profiles.` And remove the line above it, which says it will disable pass_through